packages/shared/src/utils/validator/errors.ts

```diff
@@ -1,9 +1,11 @@
 class Errors {
+    errors: { [key: string]: Array<string> };
```

Comment:
```suggestion
    errors: { [key: string]: string[] };
```
@rostislav-deriv I think we don't use `Array` generic in our codebase 🤔